Content.Server/IdentityManagement/IdentitySystem.cs

```diff
@@ -64,6 +67,16 @@ private void OnMapInit(EntityUid uid, IdentityComponent component, MapInitEvent
         _container.Insert(ident, component.IdentityEntitySlot);
     }
 
+    /// <summary>
+    ///     When the identity of a person is changed, searches the criminal records to see if the name of the new identity
+    ///     has a record. If the new name has a criminal status attached to it, the person will get the criminal status
+    ///     until they change identity again.
+    /// </summary>
+    private void OnIdentityChanged(Entity<IdentityComponent> ent, ref IdentityChangedEvent args)
+    {
+        _criminalRecordsConsole.CheckNewIdentity(ent);
+    }
```

Comment:
This event handler doesn't really make sense because you're subscribing to an event that the system itself raises. Just make it a method call and avoid the overhead.